docs/reference/tab-widgets/snapshot-repo.asciidoc

```diff
@@ -0,0 +1,15 @@
+// tag::cloud[]
+{ess} supports repositories for the following providers:
+
+* {cloud}/ec-gcs-snapshotting.html[Google Cloud Storage (GCS)]
+* {cloud}/ec-azure-snapshotting.html[Azure Blob Storage]
+* {cloud}/ec-aws-custom-repository.html[Amazon Web Services (AWS)]
+// end::cloud[]
+
+// tag::self-managed[]
+Use one of our official snapshot repository plugins:
+
+* {plugins}/repository-gcs.html[Google Cloud Storage (GCS)]
+* {plugins}/repository-azure.html[Azure Blob Storage]
+* {plugins}/repository-s3.html[Amazon Web Services (AWS)]
```

Comment:
We provide support for more repository types in the self-managed case, in particular also HDFS and shared filesystem repositories, see also the `searchable-snapshots-repository-types` section in the searchable snapshot docs which this section somewhat duplicates.

Reply:
Thanks for pointing out this duplication. I've replaced this section with a `include::...` that reuses the content from that page.